.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
```

Comment:
I think we are still using go 1.13 in go.mod
https://github.com/keikoproj/instance-manager/blob/master/go.mod#L3

Not sure if we want to keep these the same for now